go/tools/bazel/runfiles/BUILD.bazel

```diff
@@ -0,0 +1,19 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
```

Comment:
I think these sources should be folded into `//go/tools/bazel:go_default_library`.

`bazel.Runfile` should be the only function that test authors need to call directly. I don't think there's any reason to instantiate a `runfile.Resolver` outside of the `bazel` package. I'd prefer to avoid introducing public APIs if at all possible, so I think it makes sense for the `Resolver` interface to be private, folded into the `bazel` package.

Reply:
Done.